src/actions/attack.cpp

```diff
@@ -317,7 +317,7 @@ battle_context_unit_stats::battle_context_unit_stats(const unit_type* u_type,
 	int base_damage = weapon->modified_damage(backstab_pos);
 	int damage_multiplier = 100;
 	damage_multiplier
-			+= generic_combat_modifier(lawful_bonus, u_type->alignment(), u_type->musthave_status("fearless"));
+			+= generic_combat_modifier(lawful_bonus, u_type->alignment(), u_type->musthave_status("fearless"), 0);
```

Comment:
Why pass 0 here?

Reply:
This is temporary by-pass until I make deeper modifications to this part of the AI recruitment detail. As of today, the AI over recruits liminal, overestimating them. With liminal as a bonus, it tends to underestimate them, but he error in strength estimation is halved in standard schedule.

Reply:
Pinging @mattsc on this. If nothing else, it's something to add to the todo list?